examples/import/foo.arrai

```diff
@@ -0,0 +1 @@
+//{/examples/import/bar}
```

Comment:
The bug also mentions an issue when you have a module import followed by a relative import. Can we put both in the same file? (maybe a third file like `//{/examples/import/module_import} + //{./relative_import}`)